packages/Collapsible/src/Collapsible.styles.js

```diff
@@ -0,0 +1,48 @@
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const collapsibleStyles = props => `
+  box-sizing: border-box;
+  color: ${tokens.color.black};
+  display: block;
+  ${stylers.fontSize()};
+  line-height: 24px;
+  width: 100%;
+
+  > .raw-button {
+    border-radius: ${tokens.borderRadius};
+  }
+
+  .collapsible__body {
+    display: none;
+    width: 100%;
+    ${props.isCollapsed ? "display: none;" : "display: block;"}
+
+    p {
```

Comment:
☹️ Can we avoid this? Or at least be more specific (`> p:first-child { margin-top:0; }`)?

Reply:
Hm why do you want to avoid this?

Reply:
Because it will affect all the `<p>` elements in the Collapsible's content.  This shouldn't be the concern of the Collapsible, and the increased CSS specificity will make it annoying to undo if we want to in some situations.  Essentially, the Collapsible shouldn't know or care about what goes into it.